kolibri/core/assets/src/actions.js

```diff
@@ -0,0 +1,21 @@
+
+const UserKinds = require('./constants').UserKinds;
+
+function logIn(store) {
+  store.dispatch('CORE_SET_SESSION', {
+    kind: UserKinds.ADMIN,
+    facility_id: '1',
+    user_id: '2',
+    username: 'starchy52',
+    fullname: 'Mr. Potato Head',
+  });
```

Comment:
The ID doesn't need to go into Vuex, it can just be hard-coded (as "current", as discussed) into the actions.